pyriemann/utils/viz.py

```diff
@@ -1,44 +1,57 @@
 """Helpers for vizualization."""
-import seaborn as sns
 import pandas as pd
 import numpy as np
-
 from sklearn.metrics import confusion_matrix
 from pyriemann.embedding import Embedding
-import matplotlib.pyplot as plt
 
 
-def plot_confusion_matrix(targets, predictions, target_names,
-                          title='Confusion matrix', cmap="Blues"):
-    """Plot Confusion Matrix."""
+def plot_confusion_matrix(
```

Comment:
when you deprecate a function you should say in which version is will be removed.

it should also warn with a DeprecationWarning when the function is called.

You can borrow the deprecated decorator in mne or sklearn

🙏